transforms/setup-config-with-workspaces/__testfixtures__/basic.output.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+module.exports = function (defaults) {
+  let app = new EmberApp(defaults, {
+    'ember-cli-pemberly-i18n': i18nConfig,
+
+    emberHighCharts: {
+      includeHighCharts: true,
+      includeHighStock: false,
+      includeHighChartsMore: true,
+      includeHighCharts3D: true,
+      includeModules: ['solid-gauge'],
+    },
+
+    babel: {
+      plugins: [[require.resolve('babel-plugin-ember-test-metadata'), {
+        enabled: !!process.env.BABEL_TEST_METADATA,
+        packageName: defaults.project.pkg.name,
+        isUsingEmbroider: !!process.env.EMBROIDER,
+        projectRoot: '../..',
+      }]],
```

Comment:
Can we add support and test for the use case where the config exists but needs the `projectRoot` option added? For example, an app may get converted to use workspaces, but the owners never updated the config.

Before:
```js
plugins: [[require.resolve('babel-plugin-ember-test-metadata'), {
        enabled: !!process.env.BABEL_TEST_METADATA,
        packageName: defaults.project.pkg.name,
      }]],
```

After:
```js
plugins: [[require.resolve('babel-plugin-ember-test-metadata'), {
        enabled: !!process.env.BABEL_TEST_METADATA,
        packageName: defaults.project.pkg.name,
        projectRoot: '../..',
      }]],
```

WDYT?

Reply:
Yeah, no problem. I'm lazy to add it before because we have some similar tests for this in `setup-config` transform.